lib/build.py

```diff
@@ -360,7 +360,14 @@ def __init__(self, *args):
          self.dst = paths[-1]
       else:
          assert (ch.tree_child(self.tree, "copy_list") is not None)
-         self.unsupported_yet_fatal("list form", 784)
+         paths = [variables_sub(i, env.env_build)
+                  for i in ch.tree_child_terminals(self.tree, "copy_list",
+                                                   "STRING_QUOTED")]
+         self.srcs = paths[:-1]
+         for i in range(len(self.srcs)):
+            self.srcs[i] = self.srcs[i][1:-1]
```

Comment:
1. Comment what you are doing here.
2. It seems like removing the quotation marks should be the grammar's job. That's out of scope for this PR, but can you create an issue for it?